latest/index.bs

```diff
@@ -205,25 +207,69 @@ Metadata {#metadata}
 The various `.zattrs` files throughout the above array hierarchy may contain metadata
 keys as specified below for discovering certain types of data, especially images.
 
+"axes" metadata {#axes-md}
+--------------------------
+
+"axes" describes the dimensions of a physical coordinate space. It is a list of dictionaries, where each dictionary describes an dimension (axis) and:
+- MUST contain the field "name" that gives the name for this dimension. The values MUST be unique across all "name" fields.
```

Comment:
Although the `type` of axes is discussed below, it would probably be useful to include here, e.g:
 - MUST contain the field "type" with a value of "space", "time" or "channel". See [[#multiscale-md]] for more details.

Reply:
Thanks for catching this @will-moore. I think `type` got lost here in some edits. I have added it again.